README.md

```diff
@@ -1,8 +1,28 @@
-# Getting Started with Create React App
+# Welcome to South China Normal University Library
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+***This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).***
 
-## Available Scripts
+## About
+
+We are the library management platform of South China Normal University, where you can get information about the library's activities. 
+
+## How to run this app
+
+Firs clone the app with: 
+
+```bash
+git clone https://github.com/scnu-lib/scnu-lib-fe-remake.git
+```
+Now change into the app's directory and install the dependencies
+
+```bash
+cd scnu-lib-fe-remake
+```
+To install dependencies, type:
+
+```bash
+npm install
```

Comment:
Do we use `yarn` or `npm?`  There is a` yarn.lock`.Is there a problem with the current version mixing these two things together?